aws/sagemaker/large-model-inference/sample-llm/rollingbatch_deploy_llama2-13b-gptq.ipynb

```diff
@@ -0,0 +1,250 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ec9ac353",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "import sagemaker\n",
+    "from sagemaker import Model, image_uris, serializers, deserializers\n",
+    "\n",
+    "role = sagemaker.get_execution_role()  # execution role for the endpoint\n",
+    "sess = sagemaker.session.Session()  # sagemaker session for interacting with different AWS APIs\n",
+    "region = sess._region_name  # region name of the current SageMaker Studio environment\n",
+    "account_id = sess.account_id()  # account_id of the current SageMaker Studio environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "81deac79",
+   "metadata": {},
+   "source": [
+    "## Step 2: Start preparing model artifacts\n",
+    "In LMI contianer, we expect some artifacts to help setting up the model\n",
+    "- serving.properties (required): Defines the model server settings\n",
+    "- model.py (optional): A python file to define the core inference logic\n",
+    "- requirements.txt (optional): Any additional pip wheel need to install\n",
+    "\n",
+    "**Note** The original codeGen 2.5 model will not work only if [this change](https://huggingface.co/Salesforce/codegen25-7b-multi/discussions/8/files) has applied. Please apply to your own tokenizer.py when you downloaded the model or wait SalesForce to merge it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b011bf5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile serving.properties\n",
+    "engine=MPI\n",
+    "option.model_id=TheBloke/Llama-2-13B-GPTQ\n",
+    "option.tensor_parallel_degree=1\n",
+    "option.max_rolling_batch_size=32\n",
+    "option.rolling_batch=auto\n",
+    "option.quantize=gptq"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0142973",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sh\n",
+    "mkdir mymodel\n",
+    "mv serving.properties mymodel/\n",
+    "tar czvf mymodel.tar.gz mymodel/\n",
+    "rm -rf mymodel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e58cf33",
+   "metadata": {},
+   "source": [
+    "## Step 3: Start building SageMaker endpoint\n",
+    "In this step, we will build SageMaker endpoint from scratch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d955679",
+   "metadata": {},
+   "source": [
+    "### Getting the container image URI\n",
+    "\n",
+    "[Large Model Inference available DLC](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a174b36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_uri = image_uris.retrieve(\n",
+    "        framework=\"djl-deepspeed\",\n",
+    "        region=sess.boto_session.region_name,\n",
+    "        version=\"0.24.0\"\n",
```

Comment:
@lanking520  Getting this error when running this notebook:
```
ValueError: Unsupported djl-deepspeed version: 0.24.0. You may need to upgrade your SDK version (pip install -U sagemaker) for newer djl-deepspeed versions. Supported djl-deepspeed version(s): 0.23.0, 0.22.1, 0.21.0, 0.20.0, 0.19.0.
```
This is the sagemaker pkg version:
```
sagemaker                     2.192.1
sagemaker-pyspark             1.4.5
```

Reply:
cc: @chirag-orbittec